CriticalMass/MapViewController.swift

```diff
@@ -254,10 +254,15 @@ extension MapViewController: MKMapViewDelegate {
         // TODO: Remove workaround when target > iOS10 since it does not seem to work with the MKMapView+Register extension
         if annotation is CriticalMassAnnotation {
             if #available(iOS 11.0, *) {
-                return mapView.dequeueReusableAnnotationView(
+                let view = mapView.dequeueReusableAnnotationView(
                     withIdentifier: CMMarkerAnnotationView.reuseIdentifier,
                     for: annotation
                 )
+                if #available(iOS 13.0, *) {
+                    let interaction = UIContextMenuInteraction(delegate: self)
+                    view.addInteraction(interaction)
+                }
```

Comment:
How about adding the Interaktion inside CMMarkerAnnotationView ?

Reply:
Possible but to me this is controller code. But I don’t have a strong opinion about it 🤷‍♂️

Reply:
Oh right. CMMarkerAnnotationController seems to be right class for that

Reply:
I meant ViewController code. moved it in the view now